### ISSUE TITLE
How can i get each label's coordinates on segmentation map?

### ISSUE THREAD

**Issue by Baek2back** [bonlime/keras-deeplab-v3-plus#124]
```python
import numpy as np
from PIL import Image
from matplotlib import pyplot as plt

from model import Deeplabv3

# Generates labels using most basic setup.  Supports various image sizes.  Returns image labels in same format
# as original image.  Normalization matches MobileNetV2

trained_image_width=512 
mean_subtraction_value=127.5
image = np.array(Image.open('imgs/image1.jpg'))

# resize to max dimension of images from training dataset
w, h, _ = image.shape
ratio = float(trained_image_width) / np.max([w, h])
resized_image = np.array(Image.fromarray(image.astype('uint8')).resize((int(ratio * h), int(ratio * w))))

# apply normalization for trained dataset images
resized_image = (resized_image / mean_subtraction_value) - 1.

# pad array to square image to match training images
pad_x = int(trained_image_width - resized_image.shape[0])
pad_y = int(trained_image_width - resized_image.shape[1])
resized_image = np.pad(resized_image, ((0, pad_x), (0, pad_y), (0, 0)), mode='constant')

# make prediction
deeplab_model = Deeplabv3()
res = deeplab_model.predict(np.expand_dims(resized_image, 0))
labels = np.argmax(res.squeeze(), -1)

# remove padding and resize back to original image
if pad_x > 0:
    labels = labels[:-pad_x]
if pad_y > 0:
    labels = labels[:, :-pad_y]
labels = np.array(Image.fromarray(labels.astype('uint8')).resize((h, w)))

plt.imshow(labels)
plt.show()
```
I run this code and get segmentation map
But, I want to get result like [https://github.com/bonlime/keras-deeplab-v3-plus/blob/master/imgs/seg_results2.png](url) and each label's coordinates on image.
How can i do that?

**Comment by bonlime** [comment]
check original TF Deeplab implementation, they have code to turn segmentation maps to that kind of images. As far as I remember, it was in one of the ipynb files

**Comment by Baek2back** [comment]
Thanks a lot. I solved that!

**Comment by praisethemoon** [comment]
Hey @Baek2back, could please share how you managed to solve that? I am having the same issue.

**Comment by Baek2back** [comment]
@praisethemoon 
I found the answer how to obtain result like [https://github.com/bonlime/keras-deeplab-v3-plus/blob/master/imgs/seg_results2.png](url) at [https://colab.research.google.com/github/tensorflow/models/blob/master/research/deeplab/deeplab_demo.ipynb](url)

**Comment by praisethemoon** [comment]
@Baek2back Much appreciated! 🙏
